src/Core/Framework/Adapter/Twig/TokenParser/EmbedTokenParser.php

```diff
@@ -0,0 +1,77 @@
+<?php declare(strict_types=1);
+
+namespace Shopware\Core\Framework\Adapter\Twig\TokenParser;
+
+use Shopware\Core\Framework\Adapter\Twig\TemplateFinderInterface;
+use Shopware\Core\Framework\Log\Package;
+use Twig\Node\EmbedNode;
+use Twig\Node\Expression\ConstantExpression;
+use Twig\Node\Expression\NameExpression;
+use Twig\Node\Expression\Variable\ContextVariable;
+use Twig\Node\Node;
+use Twig\Token;
+
+/**
+ * @see \Twig\TokenParser\EmbedTokenParser
+ */
+#[Package('framework')]
+final class EmbedTokenParser extends \Twig\TokenParser\IncludeTokenParser
```

Comment:
Should we mark all of those classes as internal? Just to be free in the future to react to changes, this should be the case for everything in adapter/twig

Reply:
Done :white_check_mark: